src/api/ui/views/table.css

```diff
@@ -31,7 +31,7 @@
     max-width: 100%;
 }
 
-.datacore-table ul,
+.datacore-table ul:not(.contains-task-list),
```

Comment:
It seems strange to have an explicit carve out for task lists here - what is this for?

Reply:
task lists are also `ul` elements under the hood, which have their own `margin-block-start` and `margin-block-end` rules that conflict with what you've written here